assets/src/story-embed-block/block/save.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import StoryPlayer from './storyPlayer';
+
+/**
+ * The block's save function (pure).
+ *
+ * Represents a cached copy of the block’s content to be shown in case
+ * the plugin is disabled.
+ *
+ * The server-side 'render_callback' is used to override this on page load.
+ *
+ * @see https://developer.wordpress.org/block-editor/developers/block-api/block-edit-save/
+ *
+ * @param {Object} props Props.
+ * @param {Object} props.attributes Block attributes.
+ * @return {null|*} Rendered block.
+ */
+function save({ attributes }) {
+  const { url, title, poster, width, height, align = 'none' } = attributes;
```

Comment:
Hmm, does `save` do anything at all if it's a dynamic block rendering the output? As far as I remember it was just ignored in this case.

Reply:
Yes. As per the doc block above:

> Represents a cached copy of the block’s content to be shown in case the plugin is disabled.
> 
> The server-side 'render_callback' is used to override this on page load.

This use case is mentioned in the handbook: https://developer.wordpress.org/block-editor/developers/block-api/block-edit-save/

Reply:
Nice, thanks for the info! 👍